docs/codeql/reusables/query-help-overview.rst

```diff
@@ -0,0 +1,5 @@
+Visit the articles below to see the documentation for the queries included in the following query suites:
+
+- ``code-scanning``: queries run by default in code scanning.
```

Comment:
```suggestion
- ``code-scanning``: queries run by default in CodeQL code scanning on GitHub.
```

Plus maybe a link to [GitHub docs](https://docs.github.com/github/finding-security-vulnerabilities-and-errors-in-your-code/about-code-scanning)? 


Perhaps not necessary at all, but it could help people who aren't familiar with the "code scanning" feature on GitHub.